apps/console/src/features/identity-providers/components/forms/authenticators/fido-authenticator-form.tsx

```diff
@@ -0,0 +1,270 @@
+                        ? JSON.parse(value.value)
+                        : value.value
+                }
+            };
+
+            resolvedInitialValues = {
+                ...resolvedInitialValues,
+                [moderatedName]: ( value.value === "true" || value.value === "false" )
+                    ? JSON.parse(value.value)
+                    : value.value
+            };
+            
+        });
+
+        setFormFields(resolvedFormFields);
+        setInitialValues(resolvedInitialValues);
+    }, [ originalInitialValues ]);
+
+    /**
+     * Prepare form values for submitting.
+     *
+     * @param values - Form values.
+     * @returns Sanitized form values.
+     */
+    const getUpdatedConfigurations = (values: FIDOAuthenticatorFormInitialValuesInterface)
+        : CommonAuthenticatorFormInitialValuesInterface => {
+
+        const properties: CommonPluggableComponentPropertyInterface[] = [];
+
+        for (const [ name, value ] of Object.entries(values)) {
+            if (name !== undefined) {
+
+                const moderatedName: string = name.replace(/_/g, ".");
+
+                properties.push({
+                    name: moderatedName,
+                    value: isBoolean(value) ? value.toString() : value
+                });
+            }
+        }
+
+        return {
+            ...originalInitialValues,
+            properties
+        };
+    };
+
+    return (
+        <Form
+            id={ FORM_ID }
+            uncontrolledForm={ false }
+            onSubmit={ (values: FIDOAuthenticatorFormInitialValuesInterface) => {
+                onSubmit(getUpdatedConfigurations(values as FIDOAuthenticatorFormInitialValuesInterface));
+            } }
+            initialValues={ initialValues }
+        >
+            <Field.Checkbox
+                ariaLabel="Allow passkey progressive enrollment"
+                name="FIDO_EnablePasskeyProgressiveEnrollment"
+                label={ "Allow passkey progressive enrollment" }
+                hint={ "Please clear this checkbox to disable passkey progressive enrollment." }
+                readOnly={ readOnly }
+                width={ 12 }
+                data-testid={ `${ testId }-enable-passkey-progressive-enrollment` }
+            />
+            <Field.Checkbox
+                ariaLabel="Allow passkey usernameless authentication"
+                name="FIDO_EnableUsernamelessAuthentication"
+                label={ "Allow passkey usernameless authentication" }
+                hint={ "Please clear this checkbox to disable usernameless authentication." }
+                readOnly={ readOnly }
+                width={ 12 }
+                data-testid={ `${ testId }-enable-passkey-usernameless-authentication` }
+            />
+    
+            <Field.Button
+                form={ FORM_ID }
+                size="small"
+                buttonType="primary_btn"
+                ariaLabel="FIDO authenticator update button"
```

Comment:
Let's try to use i18n for the aria labels too

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/4156/commits/016d2dbfe86af4ef44d585dd2ef6e246e6ad0372